### PR TITLE
K-Means - Base

### PR DESCRIPTION
**Pull request by SatYu26**
Solution for [issue#10](https://github.com/Algo-Phantoms/Algo-ScriptML/issues/10) and [issue#6](https://github.com/Algo-Phantoms/Algo-ScriptML/issues/6) (partially).

### CODE REVIEW

File: K-Means/kmeans.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+import random
+from sklearn.cluster import KMeans
+import seaborn as sns
+import numpy as np
+import pandas as pd
```
Comment (ashwani-rathee) [inline K-Means/kmeans.py:6]: Pandas has not been used in the python file,so you can remove it

File: K-Means/kmeans.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+class CustomKMeans:
+
+    def __init__(self, n_clusters=3, max_iters=20):
+        self.n_clusters = n_clusters
+        self.max_iters = max_iters
+
+    def fit(self, X):
+
+        clusters = []
+        for i in range(self.n_clusters):
```
Comment (ashwani-rathee) [inline K-Means/kmeans.py:60]: Here i is not used in the loop ,so it's another linting issue

File: K-Means/kmeans.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        self.center = center
+        self.points = []
+
+    def distance(self, point):
+        return np.sqrt(np.sum((point - self.center) ** 2))
+
+
+class CustomKMeans:
+
+    def __init__(self, n_clusters=3, max_iters=20):
```
Comment (ashwani-rathee) [inline K-Means/kmeans.py:53]: You need to have docstring in init def according to guidelines

**Comment by SatYu26** [comment]
@ashwani-rathee is it good now?

**Comment by ashwani-rathee** [comment]
@SatYu26 It's clear from my side,let's wait for a update from our admin@geekquad

**Comment by SatYu26** [comment]
> @SatYu26 It's clear from my side,let's wait for a update from our admin @geekquad

Sure sir. Thank you!

**Review by geekquad — APPROVED** [review]

**Review by geekquad — APPROVED** [review]
Amazing work @SatYu26. Keep coding :)

**Comment by SatYu26** [comment]
> Amazing work @SatYu26. Keep coding :)

Thank you sir.!!

**Pull request merged**